README.md

```diff
@@ -65,6 +64,12 @@ A project for Nimble Go Internal Certification on Web
   {{ render_icon "iconName", "additional class names" }}
   ```
 
+## Background Tasks
+
+  Use Beego's [Task Module](https://beego.me/docs/module/task.md) to schedule and process background tasks.
+  The scraping task is scheduled to run at the beginning of every minute.
```

Comment:
I am not sure why 1 minute delay between each keyword task/job. If user upload a CSV which contains 60 keys words, then the user need wait 60 minutes to see all keywords crawling result. I think 1/2 second delay is enough. What do you think? or there is some limitation in the Beego's `task` module?